cmd/proxy/actions/app.go

```diff
@@ -22,38 +20,33 @@ const Service = "proxy"
 // App is where all routes and middleware for the proxy
 // should be defined. This is the nerve center of your
 // application.
-func App(conf *config.Config) (http.Handler, error) {
-	// ENV is used to help switch settings based on where the
-	// application is being run. Default is "development".
-	ENV := conf.GoEnv
-
+func App(logger *log.Logger, conf *config.Config) (http.Handler, error) {
 	if conf.GithubToken != "" {
 		if conf.NETRCPath != "" {
-			fmt.Println("Cannot provide both GithubToken and NETRCPath. Only provide one.")
-			os.Exit(1)
+			return nil, fmt.Errorf("cannot provide both GithubToken and NETRCPath")
 		}
 
-		netrcFromToken(conf.GithubToken)
+		if err := netrcFromToken(conf.GithubToken); err != nil {
+			return nil, fmt.Errorf("netrcFromToken error: %w", err)
```

Comment:
```suggestion
			return nil, fmt.Errorf("creating .netrc from github token: %w", err)
```
Users will see these errors via the logger. Just mentioning the function is not generally helpful, as users dont know what that is.
As a bonus, consider that the log that is seen with be `Could not create app` or something like that, so it does not really need the error to also start with `could not`.